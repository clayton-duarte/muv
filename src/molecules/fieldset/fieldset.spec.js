import React from 'react';

import Fieldset from '.';

it('should render a fieldset', () => {
  const wrapper = mount(
    <Fieldset />,
  );
  expect(wrapper).toMatchSnapshot();
});
