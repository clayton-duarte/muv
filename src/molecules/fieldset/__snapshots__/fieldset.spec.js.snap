// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a fieldset 1`] = `
<Fieldset
  disabled={false}
  label="label"
  success={false}
  warn={false}
>
  <styled.fieldset
    theme={
      Object {
        "animation": Object {
          "bounce": "
  @keyframes bounce {
    from {
      bottom: 0;
    }
    25% {bottom: .5rem;}
    to {
      bottom: 0
    }
  }
  animation: bounce .3s ease
",
          "fadeIn": "
  @keyframes fadeIn {
    from {
      opacity: 0;
    }
    
    to {
      opacity: 1;
    }
  }
  animation: fadeIn .3s ease
",
          "fadeOut": "
  @keyframes fadeOut {
    from {
      opacity: 1;
    }
    
    to {
      opacity: 0;
    }
  }
  animation: fadeOut .3s ease
",
          "popIn": "
  @keyframes popIn {
    from {
      transform: scale(0);
    }
    
    to {
      transform: scale(1);
    }
  }
  animation: popIn .3s ease
",
          "popOut": "
  @keyframes popOut {
    from {
      transform: scale(1);
    }
    
    to {
      transform: scale(0);
    }
  }
  animation: popOut .3s ease
",
        },
        "color": Object {
          "black": "#222222",
          "darkgray": "#515151",
          "error": "#dd3333",
          "gray": "#a9abae",
          "lightgray": "#e5e5e5",
          "overlay": "rgba(0, 0, 0, .15)",
          "primary": "#0081c8",
          "secondary": "#00245f",
          "success": "#33bb33",
          "warn": "#dd3333",
          "white": "#fefefe",
        },
        "font": Object {
          "color": "#222222",
          "family": "\\"Ubuntu\\", sans-serif",
          "size": "16px",
          "spacing": 0,
        },
        "shape": Object {
          "border": ".125rem solid rgba(0, 0, 0, .15)",
          "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
          "margin": ".25rem .125rem",
          "padding": ".75rem .875rem",
          "radius": ".5rem",
          "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
          "size": "16px",
        },
        "transition": Object {
          "time": ".3s ease",
        },
      }
    }
  >
    <StyledComponent
      forwardedClass={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-dnqmqq",
            "isStatic": false,
            "lastClassName": "gyHpke",
            "rules": Array [
              "
margin: ",
              [Function],
              ";
border: none;
padding: 0;
",
            ],
          },
          "defaultProps": Object {
            "theme": Object {
              "animation": Object {
                "bounce": "
  @keyframes bounce {
    from {
      bottom: 0;
    }
    25% {bottom: .5rem;}
    to {
      bottom: 0
    }
  }
  animation: bounce .3s ease
",
                "fadeIn": "
  @keyframes fadeIn {
    from {
      opacity: 0;
    }
    
    to {
      opacity: 1;
    }
  }
  animation: fadeIn .3s ease
",
                "fadeOut": "
  @keyframes fadeOut {
    from {
      opacity: 1;
    }
    
    to {
      opacity: 0;
    }
  }
  animation: fadeOut .3s ease
",
                "popIn": "
  @keyframes popIn {
    from {
      transform: scale(0);
    }
    
    to {
      transform: scale(1);
    }
  }
  animation: popIn .3s ease
",
                "popOut": "
  @keyframes popOut {
    from {
      transform: scale(1);
    }
    
    to {
      transform: scale(0);
    }
  }
  animation: popOut .3s ease
",
              },
              "color": Object {
                "black": "#222222",
                "darkgray": "#515151",
                "error": "#dd3333",
                "gray": "#a9abae",
                "lightgray": "#e5e5e5",
                "overlay": "rgba(0, 0, 0, .15)",
                "primary": "#0081c8",
                "secondary": "#00245f",
                "success": "#33bb33",
                "warn": "#dd3333",
                "white": "#fefefe",
              },
              "font": Object {
                "color": "#222222",
                "family": "\\"Ubuntu\\", sans-serif",
                "size": "16px",
                "spacing": 0,
              },
              "shape": Object {
                "border": ".125rem solid rgba(0, 0, 0, .15)",
                "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
                "margin": ".25rem .125rem",
                "padding": ".75rem .875rem",
                "radius": ".5rem",
                "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
                "size": "16px",
              },
              "transition": Object {
                "time": ".3s ease",
              },
            },
          },
          "displayName": "styled.fieldset",
          "render": [Function],
          "styledComponentId": "sc-dnqmqq",
          "target": "fieldset",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
      theme={
        Object {
          "animation": Object {
            "bounce": "
  @keyframes bounce {
    from {
      bottom: 0;
    }
    25% {bottom: .5rem;}
    to {
      bottom: 0
    }
  }
  animation: bounce .3s ease
",
            "fadeIn": "
  @keyframes fadeIn {
    from {
      opacity: 0;
    }
    
    to {
      opacity: 1;
    }
  }
  animation: fadeIn .3s ease
",
            "fadeOut": "
  @keyframes fadeOut {
    from {
      opacity: 1;
    }
    
    to {
      opacity: 0;
    }
  }
  animation: fadeOut .3s ease
",
            "popIn": "
  @keyframes popIn {
    from {
      transform: scale(0);
    }
    
    to {
      transform: scale(1);
    }
  }
  animation: popIn .3s ease
",
            "popOut": "
  @keyframes popOut {
    from {
      transform: scale(1);
    }
    
    to {
      transform: scale(0);
    }
  }
  animation: popOut .3s ease
",
          },
          "color": Object {
            "black": "#222222",
            "darkgray": "#515151",
            "error": "#dd3333",
            "gray": "#a9abae",
            "lightgray": "#e5e5e5",
            "overlay": "rgba(0, 0, 0, .15)",
            "primary": "#0081c8",
            "secondary": "#00245f",
            "success": "#33bb33",
            "warn": "#dd3333",
            "white": "#fefefe",
          },
          "font": Object {
            "color": "#222222",
            "family": "\\"Ubuntu\\", sans-serif",
            "size": "16px",
            "spacing": 0,
          },
          "shape": Object {
            "border": ".125rem solid rgba(0, 0, 0, .15)",
            "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
            "margin": ".25rem .125rem",
            "padding": ".75rem .875rem",
            "radius": ".5rem",
            "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
            "size": "16px",
          },
          "transition": Object {
            "time": ".3s ease",
          },
        }
      }
    >
      <fieldset
        className="sc-dnqmqq gyHpke"
      >
        <Text
          align="left"
          isLabel={true}
          small={false}
          strong={false}
          transform="capitalize"
          type=""
        >
          <styled.p
            align="left"
            isLabel={true}
            small={false}
            strong={false}
            theme={
              Object {
                "animation": Object {
                  "bounce": "
  @keyframes bounce {
    from {
      bottom: 0;
    }
    25% {bottom: .5rem;}
    to {
      bottom: 0
    }
  }
  animation: bounce .3s ease
",
                  "fadeIn": "
  @keyframes fadeIn {
    from {
      opacity: 0;
    }
    
    to {
      opacity: 1;
    }
  }
  animation: fadeIn .3s ease
",
                  "fadeOut": "
  @keyframes fadeOut {
    from {
      opacity: 1;
    }
    
    to {
      opacity: 0;
    }
  }
  animation: fadeOut .3s ease
",
                  "popIn": "
  @keyframes popIn {
    from {
      transform: scale(0);
    }
    
    to {
      transform: scale(1);
    }
  }
  animation: popIn .3s ease
",
                  "popOut": "
  @keyframes popOut {
    from {
      transform: scale(1);
    }
    
    to {
      transform: scale(0);
    }
  }
  animation: popOut .3s ease
",
                },
                "color": Object {
                  "black": "#222222",
                  "darkgray": "#515151",
                  "error": "#dd3333",
                  "gray": "#a9abae",
                  "lightgray": "#e5e5e5",
                  "overlay": "rgba(0, 0, 0, .15)",
                  "primary": "#0081c8",
                  "secondary": "#00245f",
                  "success": "#33bb33",
                  "warn": "#dd3333",
                  "white": "#fefefe",
                },
                "font": Object {
                  "color": "#222222",
                  "family": "\\"Ubuntu\\", sans-serif",
                  "size": "16px",
                  "spacing": 0,
                },
                "shape": Object {
                  "border": ".125rem solid rgba(0, 0, 0, .15)",
                  "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
                  "margin": ".25rem .125rem",
                  "padding": ".75rem .875rem",
                  "radius": ".5rem",
                  "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
                  "size": "16px",
                },
                "transition": Object {
                  "time": ".3s ease",
                },
              }
            }
            transform="capitalize"
          >
            <StyledComponent
              align="left"
              forwardedClass={
                Object {
                  "$$typeof": Symbol(react.forward_ref),
                  "attrs": Array [],
                  "componentStyle": ComponentStyle {
                    "componentId": "sc-bwzfXH",
                    "isStatic": false,
                    "lastClassName": "dQPAAd",
                    "rules": Array [
                      "
font-size: ",
                      [Function],
                      ";
",
                      [Function],
                      "
",
                    ],
                  },
                  "defaultProps": Object {
                    "theme": Object {
                      "animation": Object {
                        "bounce": "
  @keyframes bounce {
    from {
      bottom: 0;
    }
    25% {bottom: .5rem;}
    to {
      bottom: 0
    }
  }
  animation: bounce .3s ease
",
                        "fadeIn": "
  @keyframes fadeIn {
    from {
      opacity: 0;
    }
    
    to {
      opacity: 1;
    }
  }
  animation: fadeIn .3s ease
",
                        "fadeOut": "
  @keyframes fadeOut {
    from {
      opacity: 1;
    }
    
    to {
      opacity: 0;
    }
  }
  animation: fadeOut .3s ease
",
                        "popIn": "
  @keyframes popIn {
    from {
      transform: scale(0);
    }
    
    to {
      transform: scale(1);
    }
  }
  animation: popIn .3s ease
",
                        "popOut": "
  @keyframes popOut {
    from {
      transform: scale(1);
    }
    
    to {
      transform: scale(0);
    }
  }
  animation: popOut .3s ease
",
                      },
                      "color": Object {
                        "black": "#222222",
                        "darkgray": "#515151",
                        "error": "#dd3333",
                        "gray": "#a9abae",
                        "lightgray": "#e5e5e5",
                        "overlay": "rgba(0, 0, 0, .15)",
                        "primary": "#0081c8",
                        "secondary": "#00245f",
                        "success": "#33bb33",
                        "warn": "#dd3333",
                        "white": "#fefefe",
                      },
                      "font": Object {
                        "color": "#222222",
                        "family": "\\"Ubuntu\\", sans-serif",
                        "size": "16px",
                        "spacing": 0,
                      },
                      "shape": Object {
                        "border": ".125rem solid rgba(0, 0, 0, .15)",
                        "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
                        "margin": ".25rem .125rem",
                        "padding": ".75rem .875rem",
                        "radius": ".5rem",
                        "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
                        "size": "16px",
                      },
                      "transition": Object {
                        "time": ".3s ease",
                      },
                    },
                  },
                  "displayName": "styled.p",
                  "render": [Function],
                  "styledComponentId": "sc-bwzfXH",
                  "target": "p",
                  "toString": [Function],
                  "warnTooManyClasses": [Function],
                  "withComponent": [Function],
                }
              }
              forwardedRef={null}
              isLabel={true}
              small={false}
              strong={false}
              theme={
                Object {
                  "animation": Object {
                    "bounce": "
  @keyframes bounce {
    from {
      bottom: 0;
    }
    25% {bottom: .5rem;}
    to {
      bottom: 0
    }
  }
  animation: bounce .3s ease
",
                    "fadeIn": "
  @keyframes fadeIn {
    from {
      opacity: 0;
    }
    
    to {
      opacity: 1;
    }
  }
  animation: fadeIn .3s ease
",
                    "fadeOut": "
  @keyframes fadeOut {
    from {
      opacity: 1;
    }
    
    to {
      opacity: 0;
    }
  }
  animation: fadeOut .3s ease
",
                    "popIn": "
  @keyframes popIn {
    from {
      transform: scale(0);
    }
    
    to {
      transform: scale(1);
    }
  }
  animation: popIn .3s ease
",
                    "popOut": "
  @keyframes popOut {
    from {
      transform: scale(1);
    }
    
    to {
      transform: scale(0);
    }
  }
  animation: popOut .3s ease
",
                  },
                  "color": Object {
                    "black": "#222222",
                    "darkgray": "#515151",
                    "error": "#dd3333",
                    "gray": "#a9abae",
                    "lightgray": "#e5e5e5",
                    "overlay": "rgba(0, 0, 0, .15)",
                    "primary": "#0081c8",
                    "secondary": "#00245f",
                    "success": "#33bb33",
                    "warn": "#dd3333",
                    "white": "#fefefe",
                  },
                  "font": Object {
                    "color": "#222222",
                    "family": "\\"Ubuntu\\", sans-serif",
                    "size": "16px",
                    "spacing": 0,
                  },
                  "shape": Object {
                    "border": ".125rem solid rgba(0, 0, 0, .15)",
                    "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
                    "margin": ".25rem .125rem",
                    "padding": ".75rem .875rem",
                    "radius": ".5rem",
                    "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
                    "size": "16px",
                  },
                  "transition": Object {
                    "time": ".3s ease",
                  },
                }
              }
              transform="capitalize"
            >
              <p
                className="sc-bwzfXH foUVvz"
                transform="capitalize"
              >
                label:
              </p>
            </StyledComponent>
          </styled.p>
        </Text>
        <Input
          disabled={false}
          disabledPlaceholder="disabled"
          error={false}
          placeholder="placeholder"
          success={false}
          type="text"
          warn={false}
        >
          <styled.input
            disabled={false}
            disabledPlaceholder="disabled"
            error={false}
            placeholder="placeholder"
            success={false}
            theme={
              Object {
                "animation": Object {
                  "bounce": "
  @keyframes bounce {
    from {
      bottom: 0;
    }
    25% {bottom: .5rem;}
    to {
      bottom: 0
    }
  }
  animation: bounce .3s ease
",
                  "fadeIn": "
  @keyframes fadeIn {
    from {
      opacity: 0;
    }
    
    to {
      opacity: 1;
    }
  }
  animation: fadeIn .3s ease
",
                  "fadeOut": "
  @keyframes fadeOut {
    from {
      opacity: 1;
    }
    
    to {
      opacity: 0;
    }
  }
  animation: fadeOut .3s ease
",
                  "popIn": "
  @keyframes popIn {
    from {
      transform: scale(0);
    }
    
    to {
      transform: scale(1);
    }
  }
  animation: popIn .3s ease
",
                  "popOut": "
  @keyframes popOut {
    from {
      transform: scale(1);
    }
    
    to {
      transform: scale(0);
    }
  }
  animation: popOut .3s ease
",
                },
                "color": Object {
                  "black": "#222222",
                  "darkgray": "#515151",
                  "error": "#dd3333",
                  "gray": "#a9abae",
                  "lightgray": "#e5e5e5",
                  "overlay": "rgba(0, 0, 0, .15)",
                  "primary": "#0081c8",
                  "secondary": "#00245f",
                  "success": "#33bb33",
                  "warn": "#dd3333",
                  "white": "#fefefe",
                },
                "font": Object {
                  "color": "#222222",
                  "family": "\\"Ubuntu\\", sans-serif",
                  "size": "16px",
                  "spacing": 0,
                },
                "shape": Object {
                  "border": ".125rem solid rgba(0, 0, 0, .15)",
                  "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
                  "margin": ".25rem .125rem",
                  "padding": ".75rem .875rem",
                  "radius": ".5rem",
                  "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
                  "size": "16px",
                },
                "transition": Object {
                  "time": ".3s ease",
                },
              }
            }
            type="text"
            warn={false}
          >
            <StyledComponent
              disabled={false}
              disabledPlaceholder="disabled"
              error={false}
              forwardedClass={
                Object {
                  "$$typeof": Symbol(react.forward_ref),
                  "attrs": Array [],
                  "componentStyle": ComponentStyle {
                    "componentId": "sc-bdVaJa",
                    "isStatic": false,
                    "lastClassName": "cRlcBP",
                    "rules": Array [
                      "
",
                      [Function],
                      "
&:disabled {
  background-color: ",
                      [Function],
                      ";
}
",
                    ],
                  },
                  "defaultProps": Object {
                    "theme": Object {
                      "animation": Object {
                        "bounce": "
  @keyframes bounce {
    from {
      bottom: 0;
    }
    25% {bottom: .5rem;}
    to {
      bottom: 0
    }
  }
  animation: bounce .3s ease
",
                        "fadeIn": "
  @keyframes fadeIn {
    from {
      opacity: 0;
    }
    
    to {
      opacity: 1;
    }
  }
  animation: fadeIn .3s ease
",
                        "fadeOut": "
  @keyframes fadeOut {
    from {
      opacity: 1;
    }
    
    to {
      opacity: 0;
    }
  }
  animation: fadeOut .3s ease
",
                        "popIn": "
  @keyframes popIn {
    from {
      transform: scale(0);
    }
    
    to {
      transform: scale(1);
    }
  }
  animation: popIn .3s ease
",
                        "popOut": "
  @keyframes popOut {
    from {
      transform: scale(1);
    }
    
    to {
      transform: scale(0);
    }
  }
  animation: popOut .3s ease
",
                      },
                      "color": Object {
                        "black": "#222222",
                        "darkgray": "#515151",
                        "error": "#dd3333",
                        "gray": "#a9abae",
                        "lightgray": "#e5e5e5",
                        "overlay": "rgba(0, 0, 0, .15)",
                        "primary": "#0081c8",
                        "secondary": "#00245f",
                        "success": "#33bb33",
                        "warn": "#dd3333",
                        "white": "#fefefe",
                      },
                      "font": Object {
                        "color": "#222222",
                        "family": "\\"Ubuntu\\", sans-serif",
                        "size": "16px",
                        "spacing": 0,
                      },
                      "shape": Object {
                        "border": ".125rem solid rgba(0, 0, 0, .15)",
                        "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
                        "margin": ".25rem .125rem",
                        "padding": ".75rem .875rem",
                        "radius": ".5rem",
                        "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
                        "size": "16px",
                      },
                      "transition": Object {
                        "time": ".3s ease",
                      },
                    },
                  },
                  "displayName": "styled.input",
                  "render": [Function],
                  "styledComponentId": "sc-bdVaJa",
                  "target": "input",
                  "toString": [Function],
                  "warnTooManyClasses": [Function],
                  "withComponent": [Function],
                }
              }
              forwardedRef={
                Object {
                  "current": <input
                    class="sc-bdVaJa cRlcBP"
                    placeholder="placeholder"
                    type="text"
                  />,
                }
              }
              placeholder="placeholder"
              success={false}
              theme={
                Object {
                  "animation": Object {
                    "bounce": "
  @keyframes bounce {
    from {
      bottom: 0;
    }
    25% {bottom: .5rem;}
    to {
      bottom: 0
    }
  }
  animation: bounce .3s ease
",
                    "fadeIn": "
  @keyframes fadeIn {
    from {
      opacity: 0;
    }
    
    to {
      opacity: 1;
    }
  }
  animation: fadeIn .3s ease
",
                    "fadeOut": "
  @keyframes fadeOut {
    from {
      opacity: 1;
    }
    
    to {
      opacity: 0;
    }
  }
  animation: fadeOut .3s ease
",
                    "popIn": "
  @keyframes popIn {
    from {
      transform: scale(0);
    }
    
    to {
      transform: scale(1);
    }
  }
  animation: popIn .3s ease
",
                    "popOut": "
  @keyframes popOut {
    from {
      transform: scale(1);
    }
    
    to {
      transform: scale(0);
    }
  }
  animation: popOut .3s ease
",
                  },
                  "color": Object {
                    "black": "#222222",
                    "darkgray": "#515151",
                    "error": "#dd3333",
                    "gray": "#a9abae",
                    "lightgray": "#e5e5e5",
                    "overlay": "rgba(0, 0, 0, .15)",
                    "primary": "#0081c8",
                    "secondary": "#00245f",
                    "success": "#33bb33",
                    "warn": "#dd3333",
                    "white": "#fefefe",
                  },
                  "font": Object {
                    "color": "#222222",
                    "family": "\\"Ubuntu\\", sans-serif",
                    "size": "16px",
                    "spacing": 0,
                  },
                  "shape": Object {
                    "border": ".125rem solid rgba(0, 0, 0, .15)",
                    "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
                    "margin": ".25rem .125rem",
                    "padding": ".75rem .875rem",
                    "radius": ".5rem",
                    "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
                    "size": "16px",
                  },
                  "transition": Object {
                    "time": ".3s ease",
                  },
                }
              }
              type="text"
              warn={false}
            >
              <input
                className="sc-bdVaJa cRlcBP"
                disabled={false}
                placeholder="placeholder"
                type="text"
              />
            </StyledComponent>
          </styled.input>
        </Input>
        <Text
          align="left"
          isLabel={false}
          small={true}
          strong={false}
          transform="none"
          type=""
        >
          <styled.p
            align="left"
            isLabel={false}
            small={true}
            strong={false}
            theme={
              Object {
                "animation": Object {
                  "bounce": "
  @keyframes bounce {
    from {
      bottom: 0;
    }
    25% {bottom: .5rem;}
    to {
      bottom: 0
    }
  }
  animation: bounce .3s ease
",
                  "fadeIn": "
  @keyframes fadeIn {
    from {
      opacity: 0;
    }
    
    to {
      opacity: 1;
    }
  }
  animation: fadeIn .3s ease
",
                  "fadeOut": "
  @keyframes fadeOut {
    from {
      opacity: 1;
    }
    
    to {
      opacity: 0;
    }
  }
  animation: fadeOut .3s ease
",
                  "popIn": "
  @keyframes popIn {
    from {
      transform: scale(0);
    }
    
    to {
      transform: scale(1);
    }
  }
  animation: popIn .3s ease
",
                  "popOut": "
  @keyframes popOut {
    from {
      transform: scale(1);
    }
    
    to {
      transform: scale(0);
    }
  }
  animation: popOut .3s ease
",
                },
                "color": Object {
                  "black": "#222222",
                  "darkgray": "#515151",
                  "error": "#dd3333",
                  "gray": "#a9abae",
                  "lightgray": "#e5e5e5",
                  "overlay": "rgba(0, 0, 0, .15)",
                  "primary": "#0081c8",
                  "secondary": "#00245f",
                  "success": "#33bb33",
                  "warn": "#dd3333",
                  "white": "#fefefe",
                },
                "font": Object {
                  "color": "#222222",
                  "family": "\\"Ubuntu\\", sans-serif",
                  "size": "16px",
                  "spacing": 0,
                },
                "shape": Object {
                  "border": ".125rem solid rgba(0, 0, 0, .15)",
                  "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
                  "margin": ".25rem .125rem",
                  "padding": ".75rem .875rem",
                  "radius": ".5rem",
                  "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
                  "size": "16px",
                },
                "transition": Object {
                  "time": ".3s ease",
                },
              }
            }
            transform="none"
          >
            <StyledComponent
              align="left"
              forwardedClass={
                Object {
                  "$$typeof": Symbol(react.forward_ref),
                  "attrs": Array [],
                  "componentStyle": ComponentStyle {
                    "componentId": "sc-bwzfXH",
                    "isStatic": false,
                    "lastClassName": "dQPAAd",
                    "rules": Array [
                      "
font-size: ",
                      [Function],
                      ";
",
                      [Function],
                      "
",
                    ],
                  },
                  "defaultProps": Object {
                    "theme": Object {
                      "animation": Object {
                        "bounce": "
  @keyframes bounce {
    from {
      bottom: 0;
    }
    25% {bottom: .5rem;}
    to {
      bottom: 0
    }
  }
  animation: bounce .3s ease
",
                        "fadeIn": "
  @keyframes fadeIn {
    from {
      opacity: 0;
    }
    
    to {
      opacity: 1;
    }
  }
  animation: fadeIn .3s ease
",
                        "fadeOut": "
  @keyframes fadeOut {
    from {
      opacity: 1;
    }
    
    to {
      opacity: 0;
    }
  }
  animation: fadeOut .3s ease
",
                        "popIn": "
  @keyframes popIn {
    from {
      transform: scale(0);
    }
    
    to {
      transform: scale(1);
    }
  }
  animation: popIn .3s ease
",
                        "popOut": "
  @keyframes popOut {
    from {
      transform: scale(1);
    }
    
    to {
      transform: scale(0);
    }
  }
  animation: popOut .3s ease
",
                      },
                      "color": Object {
                        "black": "#222222",
                        "darkgray": "#515151",
                        "error": "#dd3333",
                        "gray": "#a9abae",
                        "lightgray": "#e5e5e5",
                        "overlay": "rgba(0, 0, 0, .15)",
                        "primary": "#0081c8",
                        "secondary": "#00245f",
                        "success": "#33bb33",
                        "warn": "#dd3333",
                        "white": "#fefefe",
                      },
                      "font": Object {
                        "color": "#222222",
                        "family": "\\"Ubuntu\\", sans-serif",
                        "size": "16px",
                        "spacing": 0,
                      },
                      "shape": Object {
                        "border": ".125rem solid rgba(0, 0, 0, .15)",
                        "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
                        "margin": ".25rem .125rem",
                        "padding": ".75rem .875rem",
                        "radius": ".5rem",
                        "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
                        "size": "16px",
                      },
                      "transition": Object {
                        "time": ".3s ease",
                      },
                    },
                  },
                  "displayName": "styled.p",
                  "render": [Function],
                  "styledComponentId": "sc-bwzfXH",
                  "target": "p",
                  "toString": [Function],
                  "warnTooManyClasses": [Function],
                  "withComponent": [Function],
                }
              }
              forwardedRef={null}
              isLabel={false}
              small={true}
              strong={false}
              theme={
                Object {
                  "animation": Object {
                    "bounce": "
  @keyframes bounce {
    from {
      bottom: 0;
    }
    25% {bottom: .5rem;}
    to {
      bottom: 0
    }
  }
  animation: bounce .3s ease
",
                    "fadeIn": "
  @keyframes fadeIn {
    from {
      opacity: 0;
    }
    
    to {
      opacity: 1;
    }
  }
  animation: fadeIn .3s ease
",
                    "fadeOut": "
  @keyframes fadeOut {
    from {
      opacity: 1;
    }
    
    to {
      opacity: 0;
    }
  }
  animation: fadeOut .3s ease
",
                    "popIn": "
  @keyframes popIn {
    from {
      transform: scale(0);
    }
    
    to {
      transform: scale(1);
    }
  }
  animation: popIn .3s ease
",
                    "popOut": "
  @keyframes popOut {
    from {
      transform: scale(1);
    }
    
    to {
      transform: scale(0);
    }
  }
  animation: popOut .3s ease
",
                  },
                  "color": Object {
                    "black": "#222222",
                    "darkgray": "#515151",
                    "error": "#dd3333",
                    "gray": "#a9abae",
                    "lightgray": "#e5e5e5",
                    "overlay": "rgba(0, 0, 0, .15)",
                    "primary": "#0081c8",
                    "secondary": "#00245f",
                    "success": "#33bb33",
                    "warn": "#dd3333",
                    "white": "#fefefe",
                  },
                  "font": Object {
                    "color": "#222222",
                    "family": "\\"Ubuntu\\", sans-serif",
                    "size": "16px",
                    "spacing": 0,
                  },
                  "shape": Object {
                    "border": ".125rem solid rgba(0, 0, 0, .15)",
                    "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
                    "margin": ".25rem .125rem",
                    "padding": ".75rem .875rem",
                    "radius": ".5rem",
                    "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
                    "size": "16px",
                  },
                  "transition": Object {
                    "time": ".3s ease",
                  },
                }
              }
              transform="none"
            >
              <p
                className="sc-bwzfXH dQPAAd"
                transform="none"
              >
                 
              </p>
            </StyledComponent>
          </styled.p>
        </Text>
      </fieldset>
    </StyledComponent>
  </styled.fieldset>
</Fieldset>
`;
